src/AppBundle/Committee/CommitteeManager.php

```diff
@@ -2,33 +2,73 @@
 
 namespace AppBundle\Committee;
 
+use AppBundle\Entity\Adherent;
 use AppBundle\Entity\Committee;
 use AppBundle\Entity\CommitteeMembership;
-use AppBundle\Repository\AdherentRepository;
-use AppBundle\Repository\CommitteeMembershipRepository;
+use AppBundle\Geocoder\Coordinates;
+use Doctrine\Common\Persistence\ManagerRegistry;
 
 class CommitteeManager
 {
-    private $adherentRepository;
-    private $membershipRepository;
-
-    public function __construct(
-        AdherentRepository $adherentRepository,
-        CommitteeMembershipRepository $membershipRepository
-    ) {
-        $this->adherentRepository = $adherentRepository;
-        $this->membershipRepository = $membershipRepository;
+    private const COMMITTEE_PROPOSALS_COUNT = 3;
+
+    private $registry;
+
+    public function __construct(ManagerRegistry $registry)
```

Comment:
Why? The registry is a service locator. It's better to inject the repositories we need so that we can mock them.